src/app/tests/suites/TestFabricSyncBridgedNode.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test Fabric Synchronization condition on Bridged Node Device Type
```

Comment:
Just a note that CI might fail. If it does you will want to add this test name into https://github.com/project-chip/connectedhomeip/blob/master/scripts/tests/chiptest/__init__.py#L144

or one of the other lists. That way CI doesn't pick up this YAML

Reply:
Although that said. I think the PICS will already act as filter, so you might be okay